markdown/docs/user-guide.md

```diff
@@ -257,6 +257,96 @@ export default Ember.Route.extend({
 });
 ```
 
+#### Request Method
+
+You can also access the method of the request via `fastboot.request` in the `fastboot` service.
+The `method` property will return the method name (`GET`, `POST`, `PATCH`...) of the request.
+
+```javascript
+export default Ember.Route.extend({
+  fastboot: Ember.inject.service(),
+
+  model() {
+    let method = this.get('fastboot.request.method');
+    // ...
+  }
+});
+```
+
+#### Request Body
+
+On `POST`, `PUT` and `PATCH` request you will probably be interested in the body of the request.
+You can access the `body` of the request via `fastboot.request` in the `fastboot` service, but
+only of you setup a middleware to extract the body of the request in your express server.
+
+You can use `body-parser` and `fastboot-express-middleware` and to create an in-repo addon that
+contains a middleware.
+
+
+```javascript
+// your-app/lib/fastboot-middleware/index.js
```

Comment:
Ideally both. I like to have my dev env respond to the same kind of requests I receive in prod. What changes would you add to make this "universal"?